search/Dockerfile

```diff
@@ -1,11 +1,33 @@
-FROM node:18-alpine3.18 AS runtime
+FROM node:18 as build
+
 WORKDIR /app
 
-RUN apk update && apk add --no-cache openssl-dev libpq pkgconfig build-base
-COPY package.json yarn.lock ./
-RUN yarn install
+COPY package*.json ./
+
 COPY . .
-ENV HOST=0.0.0.0 \
-    PORT=3000
-EXPOSE 3000
-CMD yarn build && node ./dist/server/entry.mjs
+
+RUN yarn install
+RUN yarn run build
+
+FROM nginx:1.25.4-alpine-slim
+
+WORKDIR /usr/share/nginx/html
+
+COPY --from=build /app/dist /usr/share/nginx/html
+# below hack only seems to work if the container isn't built with corresponding vite variable
```

Comment:
I think using envsubst is significantly less hacky and the traditionally correct way to do this.